projectional/src/main/java/jetbrains/jetpad/projectional/cell/ProjectionalRoleSynchronizer.java

```diff
@@ -34,6 +34,7 @@
   <ContentT> void supportListContentKind(ContentKind<ContentT> kind, Function<ContentT, List<SourceT>> fromContent);
   void setOnLastItemDeleted(Runnable action);
   void setPlaceholderText(String text);
+  void disablePlaceholder(Runnable onLastItemDeleted);
```

Comment:
Okay, but what to do then in getOnLastItemDeleted() if it finds the placeholder is disabled? Throw exception, ignore (lose a focus) or focus at parent's home or end position  — that's all suggestions because projectional synchronizer simply does not know this. I propose forcing the client thinking of solution instead of making suggestions.